jenkins-integration/beaker/install/shared/install_deps.rb

```diff
@@ -0,0 +1,3 @@
+step "Install jq" do
+  install_package master, 'jq'
```

Comment:
When I ran this on one of our dev blades, puppetserver-perf-sut56.delivery.puppetlabs.net, I got an error at this step:

```
puppetserver-perf-sut54.delivery.puppetlabs.net 17:40:46$ yum -y  install jq
    Loaded plugins: fastestmirror
    Loading mirror speeds from cached hostfile
     * base: mirrors.usc.edu
     * extras: mirrors.cat.pdx.edu
     * updates: repo1.sea.innoscale.net
    No package jq available.
    Error: Nothing to do
```

Does a specific repo need to configured as part of the dev setup in order for this to work?